### ISSUE TITLE
Assess progress on rpg-skilltrees roadmap and update documentation

### ISSUE THREAD

**Issue by anchristensen** [uchicago-cs/chiventure#1188]
We would like to go through previous progress on the roadmap described here: https://github.com/uchicago-cs/chiventure/wiki/Skill-Trees-~-Skilltree-%22effect%22-implementation-and-roadmap and update the documentation to reflect what has been done and what is left to be done.

**Comment by anchristensen** [comment]
Update: a 2022 update was added to each section of the roadmap wiki page. We looked through the skilltrees code and implementations and determined what had been done since the wiki was written and what is left to implement/fix, and added this in an update to each section including effects, future roadmap to dos, version history, etc. Updates can be seen on the link here https://github.com/uchicago-cs/chiventure/wiki/Skill-Trees-~-Skilltree-%22effect%22-implementation-and-roadmap.

**Comment by anchristensen** [comment]
Closing statement: we added 2022 updates to the roadmap wiki page and opened some tasks based on our findings. We also started to think about which directions to begin moving towards with designing and implementing modules based on these findings. We met up over zoom to talk through everything and I typed up notes about what we discussed on a google doc before I pasted those updates on the wiki page.
